webpack.config.js

```diff
@@ -19,6 +19,7 @@ var externalModules = [
 	'oracledb',
```

Comment:
If you're going to lint this file it would make sense to also remove the special casing for it from .lintstagedrc